R/editing.R

```diff
@@ -190,7 +190,7 @@ update_object <- function(mn, pid, path, format_id = NULL, new_pid = NULL, sid =
   sysmeta <- clear_replication_policy(sysmeta)
 
   # Add packageId to metadata if the object is an xml file
-  if (grepl("^eml:\\/\\/ecoinformatics.org\\/eml", format_id)) {
+  if (grepl("^eml:\\/\\/ecoinformatics.org\\/eml|^https://eml.ecoinformatics.org/eml-2.2.0", format_id)) {
```

Comment:
we might future-proof this. Say, if we release EML 2.2.1, we'll need to update this. I think the second part of this pattern could safely be changed to `^https://eml.ecoinformatics.org` to make it work for the future.

Reply:
good idea